avogadro/core/array.h

```diff
@@ -43,15 +43,18 @@ class ArrayRefContainer
   explicit ArrayRefContainer(const size_t n,
                              const ValueType& value = ValueType())
     : m_ref(1), data(n, value)
-  {}
```

Comment:
I definitely don't like this change. It's adding a lot of un-needed linebreaks.

`AllowShortBlocksOnASingleLine: Empty`

Reply:
Ugh, I tried Empty but it does not work as expected.

I am using clang-format 20.1.6.